File: c2/cpp/c2.cpp
#include <iostream>

int main(){
    int choice;
    std::cout << "\n1. Water\n2. Coca Cola\n3. Pepsi\n4. Pepno\n5. 7Up\n";
    std::cout << "Enter your drink selection:" << std::endl;

    std::cin >> choice;
    /* switch(choice){ */
	/* case 1: */
	    /* std::cout << "You chose water." << std::endl; */
	    /* break; */
	/* case 2: */
	    /* std::cout << "You got Coca Cola" << std::endl; */
	    /* break; */
	/* case 3: */
	    /* std::cout << "Si, I will deliver" << std::endl; */
	    /* break; */
	/* case 4: */
	    /* std::cout << "Pepno, really? Sure" << std::endl; */
	    /* break; */
	/* case 5: */
	    /* std::cout << "7Up coming down" << std::endl; */
	    /* break; */
	/* default: */
	    /* std::cout << "Error. choice was not valid, here is your money back." << std::endl; */
    /* } */

    if (choice == 1){
	std::cout << "You chose water." << std::endl;
    } else if (choice == 2){
	std::cout << "You got Coca Cola" << std::endl; 
    } else if (choice == 3){
	std::cout << "Si, I will deliver" << std::endl;
    } else if (choice == 4){
	std::cout << "Pepno, really? Sure" << std::endl;
    } else if (choice == 5){
	std::cout << "7Up coming down" << std::endl;
    } else {
	std::cout << "Error. choice was not valid, here is your money back." << std::endl;	
    }

}

File: c2/java/c2.java
import java.util.Scanner;

class main{
    public static void main(String[] args){
	Scanner s = new Scanner(System.in);
	int choice = 0;
	System.out.println("\nWhat would you like to drink?");
	System.out.println("1. Water\n2. Coca Cola\n3. Pepsi\n4. Pepno\n5. 7Up\n");
	try{
	    choice = s.nextInt();
	}
	catch (java.util.InputMismatchException e){
	    System.out.println("You get Pepno now because you suck.");
	    choice = 4;
	    System.exit(0);
	}
	// switch(choice){
	//     case 1:
	// 	System.out.println("You chose water.");
	// 	break;
	//     case 2:
	// 	System.out.println("You got Coca Cola.");
	// 	break;
	//     case 3:
	// 	System.out.println("Si, I will deliver");
	// 	break;
	//     case 4:
	// 	System.out.println("Pepno, really? Sure");
	// 	break;
	//     case 5:
	// 	System.out.println("7Up coming down");
	// 	break;
	//     default:
	// 	System.out.println("Error. choice was not valid, here is your money back.");
	// 	break;
	// }
    
	// This is to complete the star challenge
	if(choice == 1){
	    System.out.println("You chose water.");
	} else if (choice == 2) {
	    System.out.println("You got Coca Cola.");
	} else if (choice == 3) {
	    System.out.println("Si, I will deliver");
	} else if (choice == 4) {
	    System.out.println("Pepno, really? Sure");
	} else if (choice == 5) {
	    System.out.println("7Up coming down");
	} else {
	    System.out.println("Error. choice was not valid, here is you money back.");
	}
    }
}
